source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,84 @@
+---
+owner_slack: "#modernisation-platform"
+title: Set Up Macie
+last_reviewed_on: 2024-04-18
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+# Introduction
+
+It is relatively easy to set up Macie for users and it needs to be done on an account by account basis. **NOTE Macie ONLY looks at S3 buckets and it will not look at databases or EC2 settings**, for example.
+
+To do this you should connect to the environment on which you want to create Macie. Note, for accounts with 4 versions (development, test, preproduction, production) you will need to create Macie on each one where you want to use macie. There is an AWS video explaining the set up and use of macie [demo](https://www.youtube.com/watch?v=8piwEQJJXdo) which you can watch.
+
+AWS also have a brief outline of the purpose of [Macie](https://docs.aws.amazon.com/macie/latest/user/what-is-macie.html). Check the documentation to to see if it is suitable.
+
+Starting Macie with Terraform is possible and the instructions are shown below. I have also listed the details of setting up jobs. it may be beneficial to set up jobs manually due to the items that can be included. The example only shows a few of the many items that can be included. Details from Terraform are shown below the example code.
+
+# Terraform code
+
+Below there is an example of how the items were set up in Example following the instructions listed here.
+
+## Example Terraform code
+
+The MemberInfrastructureAccess was amended to include "macie2:*" so no other user access is required. All users can access the Amazon Macie through the system by following the code below.
+
+One item is needed before the code below is run. This is the requirement to set up some access to the S3 buckets you want to check. It can be added to "data.tf". This is shown below in the code (based on an example from Example) and is needed to allow the likes of "data.aws_s3_bucket.bucket1.id" to work. I have shown bucket1 in the code below. 
```

Comment:
```suggestion
```

I don't think this is needed, and how they reference the S3 bucket will depend on their code, it might not be via a data call